src/github.com/getlantern/flashlight/config/config.go

```diff
@@ -309,6 +313,27 @@ func (cfg *Config) ApplyDefaults() {
 	}
 }
 
+func defaultRoundRobin() string {
+	localeTerritory, err := jibber_jabber.DetectTerritory()
+	if err != nil {
+		localeTerritory = "us"
+	}
+	log.Debugf("Locale territory: %v", localeTerritory)
+	return defaultRoundRobinForTerritory(strings.ToLower(localeTerritory))
+}
+
+// defaultDataCenter customizes the default data center depending on the user's locale.
+func defaultRoundRobinForTerritory(localeTerritory string) string {
+	datacenter := ""
+	if localeTerritory == "cn" {
```

Comment:
Yeah it'll actually generally be upper case from ```jibber_jabber.DetectTerritory()```

We lower case it in defaultRoundRobin in ```return defaultRoundRobinForTerritory(strings.ToLower(localeTerritory))``` but you're right we really should lowercase it in the method itself -- I'll make that change thanks!